tests/Microsoft.IdentityModel.Protocol.Extensions.Tests/CrossTokenTests.cs

```diff
@@ -70,6 +70,7 @@ public void CrossToken_ValidateToken()
             JwtSecurityTokenHandler.InboundClaimFilter.Add("iss");
             JwtSecurityTokenHandler.InboundClaimFilter.Add("nbf");
 
+            string invalidIssueerJwtToken = IdentityUtilities.CreateJwtToken(IdentityUtilities.InvalidIssuerAsymmetricSecurityTokenDescriptor, jwtHandler);
```

Comment:
can you change the spelling of invalidIssueerJwtToken  (remove the extra 'e') -> invalidIssuerJwtToken